code/datums/wounds/_wounds.dm

```diff
@@ -216,6 +213,8 @@
 			victim.reagents.add_reagent(/datum/reagent/determination, WOUND_DETERMINATION_SEVERE)
```

Comment:
now that determination self-metabolizes, you should remove the nometabolism trait check earlier in this proc

Reply:
good catch